src/routing/routing.component.test.tsx

```diff
@@ -9,6 +9,8 @@ import { MemoryRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import TestAuthProvider from '../authentication/testAuthProvider';
 import * as singleSpa from 'single-spa';
+import { ThemeProvider } from '@emotion/react';
```

Comment:
I'm not an expert on the difference it makes but for consistency with other unit tests, you may want to import the `ThemeProvider` from `@mui/material` instead. This may even lead to a smaller snapshot change than the currently huge one that is present!

Reply:
Sorry, I didn't even realize, my IDE must have imported it from the wrong package!